First draft basic usage

richardjgowers opened the pull request:
Does this cover enough basic usage or have I missed something obvious?

I'll check through the tutorials later I guess to see

richardjgowers committed:
First draft basic usage

kain88-de commented on papers/oliver_beckstein/oliver_beckstein.rst:
```diff
@@ -83,6 +83,83 @@ Introduction
+
+The core object in MDAnalysis is the Universe which acts as a nexus for accessing all data contained within the simulation.
+It is initialised by passing the filenames of the topology and trajectory files,
+with a multitude of different formats are supported in these roles.
```
formats ~~are~~ supported

kain88-de commented on papers/oliver_beckstein/oliver_beckstein.rst:
```diff
@@ -83,6 +83,83 @@ Introduction
+The core object in MDAnalysis is the Universe which acts as a nexus for accessing all data contained within the simulation.
+It is initialised by passing the filenames of the topology and trajectory files,
+with a multitude of different formats are supported in these roles.
+The topology acts as a description of all the particles in the system while the trajectory describes their behaviour over time.
```
> behaviour

Do we default to American English or British English?

orbeckst replied:
Let's do American "English" ;-) given that it is published in the US.

kain88-de commented on papers/oliver_beckstein/oliver_beckstein.rst:
```diff
@@ -83,6 +83,83 @@ Introduction
+  u = mda.Universe('topol.top', 'traj.trr')
+
+  # Create a selection of atoms to work with
+  ag = u.atoms.select_atoms('name Ca and not resname MET LYS')
```
name C**A** and not resname MET LYS

kain88-de commented on papers/oliver_beckstein/oliver_beckstein.rst:
```diff
@@ -83,6 +83,83 @@ Introduction
+  ag = u.select_atoms('resname SOL and around 5.0 protein')
+
+  # Select all heavy atoms in the first 20 residues
+  ag = u.select_atoms('resid 1:20 not prop mass < 10.0')
```
shouldn't that read 

resid 1:20 **and** not prop mass < 10.0

kain88-de commented on papers/oliver_beckstein/oliver_beckstein.rst:
```diff
@@ -83,6 +83,83 @@ Introduction
+
+Changing the frame of the trajectory object updates the underlying arrays that AtomGroups point to.
+In this way the positions attribute of an AtomGroup within the iteration over u.trajectory will give access to the positions at each frame.
+Through this approach only a single frame of data is present in memory at any time, allowing for large datasets to be dissected with minimal resources.
```
You should stress that this decision is one of the reasons why MDAnalysis scales very well and allows to analyze massiv datasets. I think the @mnmelo had a nice paper for that and @tylerjereddy also had some big simulations a little while ago.

kain88-de commented on papers/oliver_beckstein/oliver_beckstein.rst:
```diff
@@ -83,6 +83,83 @@ Introduction
+
+The data from MD simulations comes in the form of a trajectory which is a frame by frame description of the motion of particles in the simulation.
+The trajectory data can often reach sizes of 10s of Gb and so reading this entirely into memory would be both slow and impractical.
+Instead, it is made accessible through the trajectory attribute of the Universe.
```
It is unclear here that the trajectory attribute helps at this point.  You haven't properly introduced the trajectory yet.  I would rather first introduce our solution that we only ever load one frame into memory and that the trajectory attribute is our interface to the currently loaded frame, as extra goodie we made it behave like a list so that common python idoms for looping can be used.

kain88-de committed:
fix spelling

kain88-de committed:
fix selections

kain88-de commented:
I rebased against 2016 and fixed the spelling and selection errors. Lets see why this still doesn't want to merge cleanly.

kain88-de commented:
Ok don't know what githubs deal is this merges cleanly on the command line.

kain88-de committed:
Add citation for big system

Direct example that we scale very well.

kain88-de committed:
Respelling of last two paragraphs

Slight changes to the last paragraph to empathize that MDAnalysis
doesn't need a lot of memory by default and is suitable for the analysis
of very large systems on common laptops.

orbeckst commented:
Then by all means merge locally and then push. (Typically, GitHub automatically closes the PR but if not, do it manually).

kain88-de committed:
make code block compile

there needs to be a new empty line after `.. code-block::`

kain88-de committed:
fix sentence formating

kain88-de commented:
rebased against current 2016 branch and now it works. Great.
